edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/appsettings_agent.json

```diff
@@ -4,6 +4,7 @@
   "ManagementUri": "http://localhost:50002",
   "DeviceConnectionString": "<Edge Device Connection String>",
   "ConfigSource": "twin",
+  "ConfigSourceBindPath": "./config.json",
```

Comment:
Not needed..

Reply:
Removed

---
In reply to: [272401044](https://github.com/Azure/iotedge/pull/1050#discussion_r272401044) [](ancestors = 272401044)